src/store/builder/index.ts

```diff
@@ -302,6 +302,8 @@ export const actions: ActionTree<IBuilderState, IRootState> = {
         timeout: 3000,
         hideProgressBar: true,
       })
+
+      dispatch('undoRedo/resetHistory', null, {root: true})
```

Comment:
@object-required , could you pls confirm this is working when we do have auto save:
- from the resource viewer page (after we change a resource & being inactive for a moment)
- from the tree build Voto5 (After being inactive for a moment)